README.md

```diff
@@ -97,9 +97,9 @@ Factory method to create a new instance of a responder server
 #### listen()
 start listening for incoming requests
 
-Class: Responder
+Interface: Responder
 ---------------------------------------------------
-Component which holds original request message and create response base on it.
+Implementation of this interface holds original request message and create response base on it.
```

Comment:
well i see no problem to have this in readme file. More over it was there before